modules/sdk-api/src/bitgoAPI.ts

```diff
@@ -187,10 +187,7 @@ export class BitGoAPI implements BitGoBase {
         common.Environments[env].stellarFederationServerUrl = params.stellarFederationServerUrl;
       }
       if (
-        params.customRootURI &&
-        params.customRootURI !== Environments.prod.uri &&
-        params.customRootURI !== Environments.test.uri &&
-        params.cookiesPropagationEnabled
+        params.customRootURI && params.cookiesPropagationEnabled
```

Comment:
Let's migrate the tests from `bitgo` to `sdk-api` and update them to match this change.

Reply:
moved this test to sdk-api/test/unit